prompting/llm.py

```diff
@@ -35,15 +61,36 @@ def load_pipeline(model_id, device=None, mock=False, model_kwargs: dict = None):
     if not device.startswith("cuda"):
         bt.logging.warning("Only crazy people run this on CPU. It is not recommended.")
 
-    # Sets default model torch type in case is not defined
-    if model_kwargs is None:
-        model_kwargs = dict(torch_dtype=torch.bfloat16)
-
-    llm_pipeline = pipeline(
-        "text-generation", model=model_id, device_map=device, model_kwargs=model_kwargs
+    tokenizer = (
+        AutoTokenizer.from_pretrained(TOKENIZER_MAPPINGS[model_id])
+        if model_id in TOKENIZER_MAPPINGS
+        else None
```

Comment:
This is what we have been doing in the past, and it's not a bad assumption. There could be situations where this is not true however, so I figured this would be a nice add on. But definitely not necessary, this could be removed if you think it's overkill.

Reply:
gotchya, will remove